R/coerce_to_messydate.R

```diff
@@ -8,12 +8,24 @@
 #' though this is a work-in-progress and currently only works in English.
 #' @param x A scalar or vector of a class that can be coerced into `mdate`,
 #'   such as `Date`, `POSIXct`, `POSIXlt`, or character.
-#' @param resequence Users have the option to choose the
-#'   order for ambiguous 6 digit dates (e.g. "11-01-12"),
-#'   and to expand these dates into precise dates (i.e. YYYY-MM-DD format).
-#'   `FALSE` by default.
-#'   If `TRUE`, it prompts users to select the existing component order of ambiguous
-#'   6 digit dates, based on which the date is reordered into YY-MM-DD format
+#' @param resequence Users have the option to choose the order for
+#'   ambiguous dates with or without separators (e.g. "11-01-12" or "20112112").
+#'   `NULL` by default.
+#'   Other options include: 'dmy', 'ymd', 'mdy', 'ym', 'my' and 'interactive'
+#'   If 'dmy', dates are converted from DDMMYY format for 6 digit dates,
+#'   or DDMMYYYY format for 8 digit dates.
+#'   If 'ymd', dates are converted from YYMMDD format for 6 digit dates,
+#'   or YYYYMMDD format for 8 digit dates.
+#'   If 'mdy', dates are converted from MMDDYY format for 6 digit dates
+#'   or MMDDYYYY format for 8 digit dates.
+#'   For these three options, ambiguous dates are converted to YY-MM-DD format
+#'   for 6 digit dates, or YYYY-MM-DD format for 8 digit dates.
+#'   If 'my', ambiguous 6 digit dates are converted from MM-YYYY format
+#'   to YYYY-MM.
+#'   If 'ym', ambiguous 6 digit dates are converted to YYYY-MM format.
+#'   If 'interactive', it prompts users to select the existing
+#'   component order of ambiguous dates,
+#'   based on which the date is reordered into YY-MM-DD format
```

Comment:
Why does interactive result in a 6 digit date instead of the more precise 8 digit format?

Reply:
It does not, this was a spelling mistake. Thank you.